blog-cse/2023-06-02-content.md

```diff
@@ -0,0 +1,16 @@
+---
+title: June 2, 2023 - Content Release
+hide_table_of_contents: true
+image: https://help.sumologic.com/img/sumo-square.png
+authors:
+  - url: https://help.sumologic.com/release-notes-cse/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Within this release, modifications to the Threat Intel MATCH-S00815 Rule to include user_username associated with the 'src_Device_ip' to capture the account the threat IP authenticated with and correlate on actions by the account. Mapper modification to the Azure Sign in Log mapper so that 'properties.userAgent' is mapped to the entity field 'http_userAgent'.
```

Comment:
```suggestion
Within this release, we made modifications to the Threat Intel MATCH-S00815 Rule to include the user_username associated with the 'src_Device_ip' to capture the account the threat IP authenticated with and to correlate on actions by the account. We also made a modification to the Azure Sign in Log mapper so that 'properties.userAgent' is mapped to the entity field 'http_userAgent'.
```